.dockerignore

```
@@ -16,4 +16,4 @@ infra
 
 # To make sure that the local appsettings files are not copied when building the image locally.
 # These files are not in GitHub thanks to .gitignore
-**/*appsettings.*.json
+
```

```suggestion
**/*appsettings.*.json
```

Please revert this change. The folder can contain files like appsettings.Development.json and appsettings.Production.json that should not included in the docker image. The files can be imported using Docker volumes, see also the main README for an example.